src/litserve/server.py

```diff
@@ -256,17 +258,26 @@ def cleanup(request_buffer, uid):
 
 @asynccontextmanager
 async def lifespan(app: FastAPI):
-    app.request_queue = Queue()
     manager = Manager()
     app.request_buffer = manager.dict()
+    app.request_queue = manager.Queue()
+
+    try:
+        pickle.dumps(app.lit_api)
+    except pickle.PickleError:
+        raise ValueError(
+            "lit_api is not pickleable. Please ensure all heavy-weight operations, like model "
```

Comment:
```suggestion
            "The LitAPI instance provided to LitServer cannot be moved to a worker because it cannot be pickled. Please ensure all heavy-weight operations, like model "
```